careers/working-at-cfpb/index.html

```diff
@@ -0,0 +1,184 @@
+            </div>
+
+            <div>
+                <section class="media
+                                block
+                                block__padded-top
+                                block__border-top
+                                block__flush-bottom">
+                    <div class="media_image-container media_image-container__wide-margin">
+                        <img class="media_image u-centered-on-mobile" src="/static/img/innovate.png">
+                    </div>
+                    <div class="media_body">
+                        <h4 class="u-mb0">Innovate</h4>
+                        <p class="short-desc">
+                            Our organization embraces new ideas and technology.
+                            We are focused on continuously improving, learning,
+                            and pushing ourselves to be great.
+                        </p>
+                    </div>
+                </section>
+            </div>
+        </section>
+
+        <div class="block">
+            <p class="short-desc">As a bureau, we support these values by focusing on equal opportunity, diversity, and inclusion. We value candidates from all backgrounds, traditional and non-traditional. We believe in investing in the growth of our colleagues and in creating an organization that is accountable to and representative of the American people.</p>
+
+            <p class="short-desc">Our service to the people who work here begins with recruitment. It also includes professional development, competitive compensation and benefits, and financial wellness programs. We want our people to learn, improve, and push themselves to be great continuously.</p>
+        </div>
+
+        <div class="block
+                    block__padded-top
+                    block__border-top
+                    content-l">
+            <section class="content-l_col-1-2">
+                {% include "templates/summaries/current-openings.html" %}
+            </section>
+            <section class="content-l_col-1-2">
+                {% include "templates/summaries/job-app-process.html" %}
+            </section>
+        </div>
+
+        {% include "templates/summaries/block-social.html" %}
+
+    </div>
+
+    <aside>
+        <div class="block
+                    block__bg
+                    block__border-top
+                    block__border-right
+                    block__flush-top
+                    block__flush-bottom
+                    block__flush-sides">
+            <div class="">
+                    {# TODO: Add Office of Civil Rights URL. #}
+                    {%- import "related-links.html" as related_links -%}
+                    {{- related_links.render([
+                        [ '/blog/',
+                          'CFPB Blog' ],
+                        [ '/newsroom/',
+                          'Newsroom' ],
+                        [ '',
+                          'Office of Civil Rights' ]
```

Comment:
Office of Civil Rights it is https://github.com/cfpb/cfgov-refresh/pull/686#issuecomment-116754972